### ISSUE TITLE
Fails to build on 4.02.0

### ISSUE THREAD

**Issue by andrewray** [dbuenzli/vg#13]
```
# ocamlfind ocamldep -package js_of_ocaml.ppx -package gg -package bytes -package uchar -package result -modules src/vgr_htmlc.ml > src/vgr_htmlc.ml.depends
# + ocamlfind ocamldep -package js_of_ocaml.ppx -package gg -package bytes -package uchar -package result -modules src/vgr_htmlc.ml > src/vgr_htmlc.ml.depends
# ocamlfind: Package `js_of_ocaml.ppx' not found
```

**Comment by dbuenzli** [comment]
This is likely a missing `js_of_ocaml` constraint. Could you try something like 
```
opam install js_of_ocaml.2.6 vg
```

**Comment by andrewray** [comment]
That fails in the same way.

https://github.com/ocsigen/js_of_ocaml/blob/b92d73489c05d8616d4defb9ee0d421d2da01c52/Makefile.conf#L62

This suggests that the ppx is not available for ocaml < 4.02.2.

**Comment by dbuenzli** [comment]
Thanks for the analysis. I constrained on master and submitted a PR to the OPAM repository.
